docs/data-ai/ai/create-dataset.md

```diff
@@ -14,320 +14,310 @@ aliases:
+1. Open the **TEST** panel of any camera or vision service component to view a feed of images from the camera.
+1. Click the button marked with the camera icon to save the currently displayed image to a dataset:
+   {{< imgproc src="/components/camera/add_image_to_dataset_button.png" alt="A button marked with the outline of a camera, emphasized in red" resize="800x" style="width:500px" class="imgzoom" >}}
+1. Select an existing dataset.
+1. Click **Add** to add the image to the selected dataset.
+1. When you see a success notification that reads "Saved image to dataset", you have successfully added the image to the dataset.
 
-{{% /expand%}}
+To view images added to your dataset, go to the **DATA** page's [**DATASETS** tab](https://app.viam.com/data/datasets) in the Viam app and select your dataset.
 
-## Prerequisites
+{{% /tab %}}
+{{% tab name="Many images" %}}
 
-{{< expand "At least 10 captured images of what you want your machine to recognize." >}}
+To capture a large number of images for training an ML model, [Capture and sync image data](/data-ai/capture-data/capture-sync/) using the data management service with your camera.
 
-[Capture and sync image data](/data-ai/capture-data/capture-sync/) using the data management service.
+Viam stores the images saved by capture and sync on the [**DATA** page](https://app.viam.com/data/), but does not add the images to a dataset. Visit the **DATA** page to add images to a dataset.
 
-When training machine learning models, it is important to supply a variety of images.
-The dataset you create should represent the possible range of visual input.
-This may include capturing images of different angles, different configurations of objects and different lighting conditions.
-The more varied the provided dataset, the more accurate the resulting model becomes.
+{{< alert title="Tip" color="tip" >}}
+
+Once you have enough images, consider disabling data capture to [avoid incurring fees](https://www.viam.com/product/pricing) for capturing large amounts of training data.
+
+{{< /alert >}}
+{{% /tab %}}
+{{< /tabs >}}
 
-{{< /expand >}}
+Once you've captured enough images for training, you must annotate them to train a model.
 
-## Tips on improving model accuracy
+## Annotate images
 
-- **More data means better models:** Incorporate as much data as you practically can to improve your model’s overall performance.
-- **Include counterexamples:** Include images with and without the object you’re looking to classify.
-  This helps the model distinguish the target object from the background and reduces the chances of false positives by teaching it what the object is not.
-- **Avoid class imbalance:** Don’t train excessively on one specific type or class, make sure each category has a roughly equal number of images.
+Use the interface on the [**DATA** tab](https://app.viam.com/data/view) to label your images. Always follow best practices when you label your images:
```

Comment:
We may need to here briefly provide an explanation as to what the different annotations do. Otherwise people will read from top to bottom and tag for classification before they realize there's a second option.

Reply:
Great point. I just switched the methods to tabs to avoid this potential area of confusion. I waffled on tabs vs. sections a few times as I reworked the page, but if someone else shares this same concern, it's probably worth accounting for!